pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -100,6 +103,7 @@
   hoc2018_tutorial_mchoc_description: 'Minecraft is back for the Hour of Code with a brand new activity! Journey through Minecraft with code.'
 
   codeorg_homepage_hoc2019_dance_heading: 'Try the new <span class="highlight-word">Dance&nbsp;Party!</span>'
+  codeorg_homepage_hoc2019_dance_heading_markdown: 'Try the new **Dance&nbsp;Party!**'
```

Comment:
Will a CSS update be done later to keep the styling the same?

Reply:
[yep!](https://github.com/code-dot-org/code-dot-org/pull/33567/files#diff-556c909484abd301aca5dd727050c6adR284-R288)